tests/test_many_files.py

```diff
@@ -24,7 +24,7 @@ def check_nexus(*args, **kwargs):
         print(f"        axes: {axes.shape if axes is not None else None}")
         print(f"      signal: {signal.shape if signal is not None else None}")
         print(f"       image: {image.shape if image is not None else None}")
-    print(f"---")
+    print("---")
     return mymap
 
 
```

Comment:
You haven't converted this file away from `unittest` and the content of `FOLDERS` is problematic for other users!